cmsplugin_articles_ai/models/articles.py

```diff
@@ -187,6 +187,20 @@ def get_absolute_url(self):
         """
         return reverse("article", kwargs={"slug": self.slug})
 
+    def clone_relations(self, src_obj, dst_obj):
+        """
+        Implements cloning relations from article to another
+
+        :param src_obj: Article object where the relations are copied from
+        :param dst_obj: Article object where the relations are copied to
+        """
+        dst_obj.tags = src_obj.tags.all()
+
+        for attachment in src_obj.attachments.all():
+            attachment.pk = None
+            attachment.article = dst_obj
+            attachment.save()
```

Comment:
This `clone_relations` is called when draft article is published. Meaning that when you click  "Publish Draft" button from the article admin it runs [this](https://github.com/andersinno/django-model-publisher-ai/blob/master/publisher/models.py#L85) publish method which eventually calls `clone_relations` method. Publish method duplicates a new Article from the current draft one. This `clone_relations` should implement cloning relations to other objects. Article has m2m field to tags and `ArticleAttachment` has foreign key to Article.